src/api/procedures/__tests__/consumeAuthorizationRequests.ts

```diff
@@ -40,6 +41,7 @@ describe('consumeAuthorizationRequests procedure', () => {
     numberToU64Stub = sinon.stub(utilsConversionModule, 'numberToU64');
     booleanToBoolStub = sinon.stub(utilsConversionModule, 'booleanToBool');
     sinon.stub(utilsConversionModule, 'addressToKey');
+    sinon.stub(utilsInternalModule, 'assertFormatValid');
```

Comment:
Instead of doing this, you should change all instantiations of Account in these tests to `entityMockUtils.getAccountInstance`